src/routes/signup/+page.svelte

```diff
@@ -11,56 +15,29 @@
 <p>Please input your credentials</p>
 
 <form action="?/signup" method="POST">
-	<div>
+	<div class="content">
 		<label for="username">Name:</label>
-		<input type="text" id="username" name="username" required />
+		<TextInput placeholder="E-Mail Adress" type="text" id="username" name="username" required="{true}"></TextInput>
 	</div>
 
-	<div>
+	<div class="content">
 		<label for="password">Password:</label>
-		<input type="password" id="password" name="password" required />
+		<TextInput placeholder="Password" type="password" id="password" name="password" required="{true}"></TextInput>
 	</div>
```

Comment:
- Misleading: this sound like the user should already have some credentials.  
Rather put something like: 
  - Please create your new credentials:  
- confirm your email / password fields (just to avoid someone making "ghost accounts")